packages/gatsby/cache-dir/app.js

```diff
@@ -36,7 +36,7 @@ apiRunnerAsync(`onClientEntry`).then(() => {
       if (services.developstatusserver) {
         let isRestarting = false
         const parentSocket = io(
-          `${window.location.protocol}//${window.location.hostname}:${services.developstatusserver.port}`
+          `${window.location.protocol}//${window.location.hostname}:${window.location.port}`
```

Comment:
This fixes the issue in my testing:

```suggestion
          `http://${window.location.hostname}:${services.developstatusserver.port}`
```

The "proper" solution for this would be to add support for HTTPS to the develop status server. However, that would be a much more involved fix for what is (essentially) a friendly warning that solely breaks the non-essential "prompt to restart on gatsby-config changes" functionality.

This makes it work even when the site runs over HTTPS without any console spam using HTTP, which seems like a good enough tradeoff?